### ISSUE TITLE
Error on windows machine.

### ISSUE THREAD

**Issue by Anksus** [timlrx/tailwind-nextjs-starter-blog#194]
While navigating to the about page, some syntax error occurs. 

Error log on the browser.
```
SyntaxError: The requested module 'mdast-util-mdx' is expected to be of type CommonJS, which does not support named exports. CommonJS modules can be imported by importing the default export.
For example:
import pkg from 'mdast-util-mdx';
const {fromMarkdown, toMarkdown} = pkg;
``` 
Error log inside the terminal.
```
file:///C:/Users/Hp/Desktop/anksus.me/node_modules/xdm/lib/plugin/remark-mdx.js:14
import {fromMarkdown, toMarkdown} from 'mdast-util-mdx'
                      ^^^^^^^^^^
SyntaxError: The requested module 'mdast-util-mdx' is expected to be of type CommonJS, which does not support 
named exports. CommonJS modules can be imported by importing the default export.
For example:
import pkg from 'mdast-util-mdx';
const {fromMarkdown, toMarkdown} = pkg;
    at ModuleJob._instantiate (internal/modules/esm/module_job.js:98:21)
    at async ModuleJob.run (internal/modules/esm/module_job.js:137:5)
    at async Loader.import (internal/modules/esm/loader.js:165:24)
    at async bundleMDX (C:\Users\Hp\Desktop\anksus.me\node_modules\mdx-bundler\dist\index.js:60:27)
    at async getFileBySlug (C:\Users\Hp\Desktop\anksus.me\.next\server\pages\about.js:2426:7)
    at async getStaticProps (C:\Users\Hp\Desktop\anksus.me\.next\server\pages\about.js:5932:25)
    at async renderToHTML (C:\Users\Hp\Desktop\anksus.me\node_modules\next\dist\next-server\server\render.js:27:1780)
    at async C:\Users\Hp\Desktop\anksus.me\node_modules\next\dist\next-server\server\next-server.js:112:97    
    at async __wrapper (C:\Users\Hp\Desktop\anksus.me\node_modules\next\dist\lib\coalesced-function.js:1:330) 
    at async DevServer.renderToHTMLWithComponents (C:\Users\Hp\Desktop\anksus.me\node_modules\next\dist\next-server\server\next-server.js:137:387)
```

The same can be produced by running the project on a windows machine and navigating to the about page.

**Comment by timlrx** [comment]
I am unable to replicate this issue on a fresh clone on windows. Is this a fresh clone or are you updating from an older version of the template? 

The latest version sets the `esmExternals` flag to true so you can import ESM directly - https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/next.config.js#L11

If you upgraded only the remark / rehype package versions but did not set this flag, it will throw the error as the new packages are ESM only.

**Comment by Anksus** [comment]
Actually, my repo is on v1 and I updated all the dependencies. It doesn't even have the `esmExternals` flag. I'll update it now. Thanks

**Comment by timlrx** [comment]
Yeah, if you are on v1, you should stick with the v1 dependencies. Upgrading would break it as the new remark & rehype packages are ESM only. 

An ESM compatible version with all the packages updated is published in master (v1.1). You might need to update some other pages - see this [PR](https://github.com/timlrx/tailwind-nextjs-starter-blog/pull/191)

**Comment by Anksus** [comment]
Okay, I'll update to v1.1. Thanks for sharing the required steps.
